Pidaq/UI/Stability/stability_dash/assets/css/custom.css

```diff
@@ -0,0 +1,44 @@
+.pad-top {
```

Comment:
We (meaning I) am pretty dumb for not laying out these dash apps to all use the same assets folder so we could avoid copy pasting what little custom css we do have

Way she goes